lib/Perl/LanguageServer/DebuggerProcess.pm

```diff
@@ -149,7 +149,11 @@ sub launch
     my @inc ;
     @inc = map { ('-I', $_)} @$inc if ($inc) ;
     
-    $ENV{PLSDI_REMOTE} = '127.0.0.1:' . $self -> debug_adapter -> listen_port ;
+    my $host = $self -> debug_adapter -> listen_host;
```

Comment:
Not sure why we need this?

Reply:
Because listen_host might be a specific ip address (the case where we listen eg. 192.168.0.12 and NOT 127.0.0.1).